bdb/file.c

```diff
@@ -1013,6 +1013,7 @@ int bdb_rename_table(bdb_state_type *bdb_state, tran_type *tran, char *newname,
 {
     DB_TXN *tid = tran ? tran->tid : NULL;
     char *orig_name;
+    char *origname; /* certain sc set this, preserve */
```

Comment:
I'd choose a different variable name. `origname` and `orig_name` are quite easily confused with each other.

Reply:
Agreed